example/src/App.js

```diff
@@ -11,8 +11,10 @@ const App = () => (
     <h1>Lego editor</h1>
     <Editor
       placeholder="Testing lego editor"
-      imageUpload={file => new Promise(resolve => setTimeout(resolve, 1000))}
-      onChange={str => {
+      imageUpload={(file) =>
```

Comment:
Looks like these are prettier formats, so you should probably make sure prettier fixes and lints `example` as well as `app`.